Source/Particles/Filter/FilterFunctors.H

```diff
@@ -70,7 +70,7 @@ struct UniformFilter
         else return 0;
     }
 private:
-    const bool m_is_active = false; //! select all particles if false
+    const bool m_is_active; //! select all particles if false
```

Comment:
wow, did not know data members can be const. today I learned...

the other member does not warn?

Reply:
If they're in the initialization list, it's good! I like that.
Actually the default value didn't cause a warning (I was exploring before understanding that the warning was due to the order in the initialization list), I can set it back, not sure what is most relevant, though.

Reply:
All good, I just thought it's related. Thanks for the info!